version/version.go

```diff
@@ -1,5 +1,17 @@
 package version
 
+import (
+	"io/ioutil"
+)
+
 var (
-	Version = "0.0.1"
+	Version = getVersion()
 )
+
+func getVersion() string {
+	version, err := ioutil.ReadFile("./operator-version")
+	if err != nil {
+		return "Unknown"
+	}
+	return string(version)
```

Comment:
Let's take into account the case where the version file is something like this:
```
0.3.0 

```
You need to take only the first line, and do a `trim` to delete whitespaces around the version number.